src/ServerPrivate.cc

```diff
@@ -384,7 +386,26 @@ void ServerPrivate::CreateEntities()
 void ServerPrivate::SetupTransport()
 {
   // Advertise available worlds.
-  this->node.Advertise("/gazebo/worlds", &ServerPrivate::WorldsService, this);
+  std::string worldsService{"/gazebo/worlds"};
+  this->node.Advertise(worldsService, &ServerPrivate::WorldsService, this);
+
+  ignmsg << "Serving world names on [" << worldsService << "]" << std::endl;
+
+  // Resource path management
+  std::string addPathService{"/gazebo/add_resource_paths"};
```

Comment:
Suggestion; use `gazebo/resource_paths/add` and `gazebo/resources_paths/get`.

Reply:
I like that pattern, we should use it more often: 0fe84c4f6680296f2099166d2de43890fa295bc9